docs/frontend/preact.md

```diff
@@ -56,7 +56,7 @@ import { tagPropTypes } from '../../../components/common-prop-types';
 
 const SomeComponentUsingTags = ({ tags = [] }) => (
   <ul>
-    {tags.map(tag => (
+    {tags.map((tag) => (
```

Comment:
was this a linter change?

Reply:
Yes. There were quite a few of them in this PR, looks like lately some things slipped into `main`.